Automate CLI releases. (#592)
fauna/fauna-shell: .changeset/small-mugs-drum.md

---
"fauna-shell": patch
---

Improve export user experience.

> - Export create now allows users to provide S3 URIs as a target to export to (with the flag `--desination`) in addition to providing each part independently (`--bucket`, `--path`)

Reviewer: ```suggestion
- `fauna export create` allows you to provide a target S3 URI with the `--destination` flag. Previously, you had to provide both the `--bucket` and `--path` separately.
```

> - Export create now allows users to provide S3 URIs as a target to export to (with the flag `--desination`) in addition to providing each part independently (`--bucket`, `--path`)

Reviewer: fixed a small typo in this one, but going to merge the intended text

> - Export create now allows users to provide S3 URIs as a target to export to (with the flag `--desination`) in addition to providing each part independently (`--bucket`, `--path`)

Reviewer 2: Thanks!
